src/NuGet.Core/NuGet.LibraryModel/CentralVersionDependency.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using NuGet.Shared;
+using NuGet.Versioning;
+
+namespace NuGet.LibraryModel
+{
+    public class CentralVersionDependency : IEquatable<CentralVersionDependency>, IComparable<CentralVersionDependency>
```

Comment:
How about CentralPackageVersion? 

Specifically, I don't think the Dependency word belongs in the name.

Reply:
Updated